hedera-node/hedera-mono-service/src/main/resources/bootstrap.properties

```diff
@@ -38,12 +38,12 @@ files.throttleDefinitions=123
 hedera.firstUserEntity=1001
 hedera.realm=0
 hedera.shard=0
-ledger.id=0x03
+ledger.id=0x00
```

Comment:
Is this intentional ?

Reply:
Yes, we believe the defaults here should be mainnet configuration values.